packages/admin-panel/src/pages/resources/MapOverlaysPage.js

```diff
@@ -138,7 +140,7 @@ const renderNewMapOverlayVizButton = () => (
   </StyledLink>
 );
 
-export const MapOverlaysPage = ({ getHeaderEl, isBESAdmin, ...props }) => {
+const MapOverlaysPageComponent = ({ getHeaderEl, isVizBuilderUser, ...props }) => {
```

Comment:
Can you use `useUser()` here instead of passing in `isVizBuilderUser` via the state props? I'd love if we could remove the whole `isVizBuilderUser` from the redux state entirely 👍